scripts/generate-mocks.js

```diff
@@ -33,4 +48,9 @@ hasChanges(inputPaths).then((hasChanged) => {
   const subFolder = '/components/';
   // core input path is not needed to generate angular mocks
   new GenerateMocks().renderMocks(inputPaths.angular, outputPaths, subFolder);
-});
+}
+
+  hasChanges(inputPaths).then((hasChanged) => {
+    if (!hasChanged || !hasFlag('force')) return;
```

Comment:
I goofed up the logic in the earlier suggestion i made 🤦 
```suggestion
    if (!hasChanged && !hasFlag('force')) return;
```